source/layouts/base.erb

```diff
@@ -1,24 +1,26 @@
 <!doctype html>
-<html>
+<html lang="en-gb">
   <head>
     <%
-      the_url = URI.join('http://lrug.org/', url_for(current_page))
+      the_url = URI.join('https://lrug.org/', url_for(current_page))
```

Comment:
Our meta open graph links were using `http`, so this should speed up the initial time it takes to open a link to the page.